### ISSUE TITLE
Console error when navigating in editor

### ISSUE THREAD

**Issue by dmbostan** [WordPress/gutenberg#16894]
**Describe the bug**
Console error when navigating in editor. Uncaught TypeError: Cannot read property 'length' of undefined

**To reproduce**
1. Create 1 Columns Block with 2 children. 
2. Add 1 button in each of the columns.
3. Navigate through the structure by using the up and down keys. The error shall appear when going from up to down.

**Expected behavior**
No error.

**Screenshots**
![image](https://user-images.githubusercontent.com/5270333/62408410-f3ce9700-b5c8-11e9-86d8-15c706a248b9.png)

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser Chrome
 - Version 75.0.3770.100 

**Additional context**
- WordPress 5.2.2 running Twenty Nineteen theme.

**Comment by youknowriad** [comment]
This seems fixed for me.
